modules/installation-ibm-cloud-config-yaml.adoc

```diff
@@ -102,10 +107,109 @@ endif::openshift-origin[]
+    resourceGroupName: eu-gb-example-network-rg <6>
+    vpcName: eu-gb-example-network-1 <7>
+    controlPlaneSubnets: <8>
+      - eu-gb-example-network-1-cp-eu-gb-1
+      - eu-gb-example-network-1-cp-eu-gb-2
+      - eu-gb-example-network-1-cp-eu-gb-3
+    computeSubnets: <9>
+      - eu-gb-example-network-1-compute-eu-gb-1
+      - eu-gb-example-network-1-compute-eu-gb-2
+      - eu-gb-example-network-1-compute-eu-gb-3
+credentialsMode: Manual
+publish: External
+pullSecret: '{"auths": ...}' <1>
+ifndef::openshift-origin[]
+fips: false <10>
+sshKey: ssh-ed25519 AAAA... <11>
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+sshKey: ssh-ed25519 AAAA... <10>
+endif::openshift-origin[]
+capabilities:
+  baselineCapabilitySet: None
+  additionalEnabledCapabilities:
+  - openshift-samples
+----
+<1> Required. The installation program prompts you for this value.
+<2> If you do not provide these parameters and values, the installation program provides the default value.
+<3> The `controlPlane` section is a single mapping, but the `compute` section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Only one control plane pool is used.
+<4> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
++
+[IMPORTANT]
+====
+If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger machine types, such as `n1-standard-8`, for your machines if you disable simultaneous multithreading.
+====
+<5> The machine CIDR must contain the subnets for the compute machines and control plane machines.
+<6> The name of an existing resource group. The existing VPC and subnets should be in this resource group. The cluster is deployed to this resource group.
+<7> Specify the name of an existing VPC.
+<8> Specify the name of the existing subnets to deploy the control plane machines to. The subnets must belong to the VPC that you specified. Specify a subnet for each availability zone in the region.
+<9> Specify the name of the existing subnets to deploy the compute machines to. The subnets must belong to the VPC that you specified. Specify a subnet for each availability zone in the region.
+ifndef::openshift-origin[]
+<10> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
```

Comment:
No. FIPS is a protocol that is well known to our intended audience. Similar to CIDR, it would be weird to actually spell it out. In every conversations, people refer to the protocol as "FIPS".